assume/common/forecasts.py

```diff
@@ -15,8 +15,8 @@ class Forecaster:
     to retrieve forecasts for specific columns, availability of units, and prices of fuel types, returning
     the corresponding timeseries as pandas Series.
 
-    Args:
-    - index (pd.Series): The index of the forecasts.
+    Attributes:
+        index (pd.Series): The index of the forecasts.
```

Comment:
It is correct to have this called Args:

If you look at this example:
https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google

the def __init__ is documented in the class doc string with Args

You would use Attributes for Class-Variables which are independent of the actual instance

Reply:
So it's not double, if it is added in both class definition and init()? Then I will add it for all others too

Reply:
yes - Attributes document the class attributes and args document the arguments to the constructor.

In most cases those are quite similar - as args to the constructor are then set as an attribute, so yes you can add them

Reply:
Thanks for the explanation 👍